docs/operator-guide/cd/deploy-application-in-remote-cluster-via-irsa.md

```diff
@@ -0,0 +1,683 @@
+---
+
+title: "Deploy Application In Remote Cluster via IRSA"
+sidebar_label: "Deploy Application In Remote Cluster via IRSA"
+description: "Learn how to securely deploy applications across AWS accounts using IRSA (IAM Roles for Service Accounts) in KubeRocketCI. Step-by-step guide for EKS cross-account access configuration."
+
+---
+<!-- markdownlint-disable MD025 -->
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Deploy Application In Remote Cluster via IRSA
+
+<head>
+  <link rel="canonical" href="https://docs.kuberocketci.io/docs/operator-guide/cd/deploy-application-in-remote-cluster-via-irsa/" />
```

Comment:
Could you please remove the trailing slash from the canonical page? We no longer need it